app/dashboard/views.py

```diff
@@ -1043,15 +1043,16 @@ def scenario_visualize_cash_flow(request, scen_id):
     ft = FinancialTool(scenario.project)
     initial_loan = ft.initial_loan_table
     replacement_loan = ft.replacement_loan_table
+    custom_tariff = ft.get_tariff()
```

Comment:
The tariff should probably be calculated once and then saved to the database and retrieved. Since the algorithm runs 1000 iterations to calculate it, it is a time intensive process and it doesnt make any sense to have it run every time. I just did not get around to doing it.